tripleo-quickstart-config/roles/libvirt/setup/overcloud/templates/instackenv.json.j2

```diff
@@ -25,7 +25,7 @@
           "pm_type": "ipmi",
         {% endif %}
         "pm_user": "admin",
-        "pm_addr": "{{ undercloud_local_ip|default(undercloud_network_cidr)|nthhost(1) }}",
+        "pm_addr": "192.168.122.1",
```

Comment:
Hard-coding this seems fine for now, but there's also a bunch of logic related to this at the top of the template we can probably also remove - I can push a follow-up

Reply:
Ack, Sounds good to me